sensei-lms.php

```diff
@@ -94,7 +94,7 @@ function is_sensei_activating( $activating_plugin, $plugin ) {
 	 */
 	function Sensei() {
 		// phpcs:enable
-		return Sensei_Main::instance( array( 'version' => '3.8.1' ) );
+		return Sensei_Main::instance( array( 'version' => '3.9.0-dev' ) );
```

Comment:
Should this be changed in all the usual places?

Reply:
Doesn't hurt. Did it everywhere except stable tag in readme 843be7f88d6f3e0288d8c25972150dfa93961823